third_party/nvfuser/csrc/ir_utils.h

```diff
@@ -310,6 +310,8 @@ TORCH_CUDA_CU_API std::vector<IndexSelectOp*> getIndexSelectOps(Fusion* fusion);
 
 TORCH_CUDA_CU_API std::vector<TorchGatherOp*> getTorchGatherOps(Fusion* fusion);
 
+TORCH_CUDA_CU_API std::vector<ScatterOp*> getScatterOps(Fusion* fusion);
```

Comment:
nit: I'm not strongly against, but we shouldn't create a `getXYZOps` for every type of ops as the code is fairly trivial.